code/modules/mob/living/carbon/human/diona_gestalt.dm

```diff
@@ -14,14 +14,13 @@
 	var/datum/dionastats/DS
 
 
-
 /mob/living/carbon/human/proc/setup_gestalt()
 	composition_reagent = "nutriment"//Dionae are plants, so eating them doesn't give animal protein
 	setup_dionastats()
 	verbs += /mob/living/carbon/human/proc/check_light
 	verbs += /mob/living/carbon/human/proc/diona_split_nymph
 	verbs += /mob/living/proc/devour
-
+	species.vision_organ = "head"
```

Comment:
no.

There's a species class called diona. Set it to be `"head"` there. Not here.